libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -56,7 +56,7 @@ object MapboxMetricsReporter : MetricsReporter {
      * Set flag to determine is need to log [mapboxTelemetry] events.
      *
      * @param isDebugLoggingEnabled true to enable logging, false to disable logging
-     * @since 0.43.0
+     * @since 1.0.0
```

Comment:
Again, not fond of those. But since they are already here, we can leave them.

Reply:
Yeah, It is not very important point, but probably we need to discuss it with a team)

Reply:
I vote for removing all of them 🔥

This is like comments, it's pretty easy to forget adding or updating them and at that point is confusing to clients + also it's something that needs maintenance which means "unnecessary" (not that worth it) extra work on our side.

Reply:
Got it, I'm ok with it 👍